tests/core/_entity/test_labelled.py

```diff
@@ -101,10 +101,10 @@ def test_get_label_complex_case():
     assert cycle.get_simple_label() == scenario.cycle.name
     assert scenario.get_label() == "Today > My Name"
     assert scenario.get_simple_label() == "My Name"
-    assert pipeline_C.get_label() == "Today > pipeline_C"
-    assert pipeline_C.get_simple_label() == "pipeline_C"
-    assert pipeline_S.get_label() == "Today > My Name > pipeline_S"
-    assert pipeline_S.get_simple_label() == "pipeline_S"
+    assert "Today > PIPELINE_pipeline_C" in pipeline_C.get_label()
+    assert "PIPELINE_pipeline_C" in pipeline_C.get_simple_label()
+    assert "Today > My Name > PIPELINE_pipeline_S" in pipeline_S.get_label()
+    assert "PIPELINE_pipeline_S" in pipeline_S.get_simple_label()
```

Comment:
That's an undesired consequence...
We should avoid as much as possible to have a label that a human cannot read. So we want to avoid having an ID as label. 
By adding the name property to the Pipeline, it should solve the pb.